pkg/operator/cassandra/controller/cluster_test.go

```diff
@@ -155,7 +154,8 @@ func TestScaleUpRack(t *testing.T) {
 				if test.expectedErr {
 					t.Errorf("Expected an error, got none.")
 				} else {
-					sts, err := cc.kubeClient.AppsV1().StatefulSets(test.cluster.Namespace).
```

Comment:
This change is unnecessary.

Reply:
See above (I'll remove this change as well for now but I'll bring it back in another PR).